src/v2/Apps/Auctions/auctionsRoutes.tsx

```diff
@@ -5,34 +5,89 @@ const AuctionsApp = loadable(() => import("./AuctionsApp"), {
   resolveComponent: component => component.AuctionsAppFragmentContainer,
 })
 
+const CurrentAuctions = loadable(() => import("./Routes/CurrentAuctions"), {
+  resolveComponent: component => component.CurrentAuctionsFragmentContainer,
+})
+
+const UpcomingAuctions = loadable(() => import("./Routes/UpcomingAuctions"), {
+  resolveComponent: component => component.UpcomingAuctionsFragmentContainer,
+})
+
+const PastAuctions = loadable(() => import("./Routes/PastAuctions"), {
+  resolveComponent: component => component.PastAuctionsFragmentContainer,
+})
+
 export const auctionsRoutes = [
   {
     path: "/auctions2",
     getComponent: () => AuctionsApp,
     prepare: () => {
       AuctionsApp.preload()
     },
+
     query: graphql`
       query auctionsRoutes_AuctionsQuery {
         me {
           ...AuctionsApp_me
         }
-        currentSales: salesConnection(
-          first: 99
-          published: true
-          sort: START_AT_ASC
-        ) {
-          ...AuctionsApp_currentSales
-        }
-        pastSales: salesConnection(
-          first: 20
-          published: true
-          live: false
-          sort: START_AT_ASC
-        ) {
-          ...AuctionsApp_pastSales
-        }
       }
     `,
+    children: [
+      {
+        path: "", // represents current auctions aka /auctions/current
+        getComponent: () => CurrentAuctions,
+        prepare: () => {
+          CurrentAuctions.preload()
+        },
+        query: graphql`
+          query auctionsRoutes_Current_AuctionsQuery {
+            currentAuctions: salesConnection(
+              first: 99
+              published: true
+              sort: START_AT_ASC
+            ) {
+              ...CurrentAuctions_currentAuctions
+            }
+          }
+        `,
+      },
+      {
+        path: "upcoming",
+        getComponent: () => UpcomingAuctions,
+        prepare: () => {
+          UpcomingAuctions.preload()
+        },
+        query: graphql`
+          query auctionsRoutes_Upcoming_AuctionsQuery {
+            upcomingAuctions: salesConnection(
+              first: 99
+              published: true
+              sort: START_AT_ASC
+            ) {
+              ...UpcomingAuctions_upcomingAuctions
+            }
+          }
+        `,
+      },
+      {
+        path: "past",
+        getComponent: () => PastAuctions,
+        prepare: () => {
+          PastAuctions.preload()
+        },
+        query: graphql`
+          query auctionsRoutes_Past_AuctionsQuery {
+            pastAuctions: salesConnection(
+              first: 20
+              published: true
+              live: false
+              sort: START_AT_ASC
+            ) {
+              ...PastAuctions_pastAuctions
+            }
+          }
+        `,
+      },
+    ],
```

Comment:
This is a general route file for an app, which follows the form of our other routes files in other `v2/Apps/<AppName>` apps.

Without all the additional noise, it's pretty easy to understand. Here's a super simple and functional router app example: 

```ts
const routes = [
  {
    path: '/example',
    Component: props => {
      return (
        <div>Hello there! What page would you like to go to?</div>
        
        <nav>
          <RouterLink to='/example/page-one'>Page 1</RouterLink>
          <RouterLink to='/example/page-two'>Page 2</RouterLink>
          <RouterLink to='/example/page-three'>Page 3</RouterLink>
        </nav>
        
        // Renders out the children component routes below, if the route matches 
        <div>{props.children}</div>
      )
    },
    children: [
      {
        path: 'page-one',
        // Just a regular react component 
        Component: () => <div>You're on page one</div>
      },
      {
        path: 'page-two',
        Component: () => <div>You're on page two</div>
      },
      {
        path: 'page-three',
        Component: () => <div>You're on page three</div>
      }
    ]
  }
]
``` 

When a user clicks the `RouterLink` components (basically just regular `a` tags) it will update the url bar, and the router will see it and then look in the `children` array for a matching route, then render the appropriate component.

Reply:
Ah is the children array holding children components that were requested in the GraphQL query?

Reply:
It's best to think about the example in the comment above as being completely divorced from graphql queries and just related to routing. I totally forgot to mention this but we use a library called [Found](https://github.com/4Catalyzer/found)! If you read over those docs it should be a bit clearer. 

Basically, children means: "this top level routes has sub routes, its children", aka:
- top level route is `path: '/example',` 
- its children are `path: '/example/page-one', `path: '/example/page-two', `path: '/example/page-three'. 
- `children` also points at react's concept of children, how one can compose components: 

```tsx
const App = props => {
  return <div>{props.children}</div>
    
}

const Child = props => {
  return <div>Hi!</div>
}

React.render(<App><Child /></App>) 
```

In the same way, the top level path of the router is like the `App` component above, and when the route matches one of the paths defined in the `children` array, the child will be rendered in the `{children}` slot.

Reply:
When it comes to graphql stuff (defined by the `query` field), thats how we fetch data for our components. 

So like, revising our example a bit: 
```tsx
const ArtistRoute = props => {
  return (
    <div>The artists name is: {props.artist.name}</div>
  )
},

const routes = [
  {
    path: '/artist',
    Component: ArtistRoute
    query: graphql`
      query SomeQuery {
        artist(id: "andy-warhol") {
          name
        }
      }
    `
]
```

The query makes a request for data on an artist, and we declare what details we need. Thats then piped into the component as react props.

Reply:
Mmmkay, okay. That makes sense, thanks!